test/events-reader.spec.js

```diff
@@ -174,14 +172,16 @@ describe('onChange callback, event capture and at-least-once delivery semantics'
           return Promise.all(_.map(models, removeModelData));
         }
 
-        var initFromLastCheckpoint = function(harvesterApp, oplogDb) {
-          var query = {}, coll = oplogDb.collection('oplog.rs');
+        var initFromLastCheckpoint = function(harvesterApp) {
```

Comment:
Please tell me this code isnt in production... I'm seeing very worrying stuff here guys. This does not init from last checkpoint. It reads the most recent update.